payments/paying-for-an-order/stripe-payments.md

```diff
@@ -28,6 +28,10 @@ The Bearer token to grant access to the API
 {% endapi-method-headers %}
 
 {% api-method-body-parameters %}
+{% api-method-parameter name="options.idempotency\_key" type="string" required=false %}
```

Comment:
Is this relevant to this PR, or should it be separate?

Reply:
I think this has been added in due to a botched rebase, I'll sort it out.